kokoro/config/test/third_party_apps/release/rockylinux9.gcl

```diff
@@ -0,0 +1,7 @@
+import 'common.gcl' as common
```

Comment:
After thinking it over, I think we should defer any third_party_apps changes for later. I doubt that this test will pass as-is on rocky-linux-9 and IMO we should  not add configs for it until our goal is actually "turn on rocky-linux-8 third_party_apps_test". And I don't think that is our goal yet.

Reply:
Reverting